### ISSUE TITLE
Coturn does not create logs

### ISSUE THREAD

**Issue by SDCRoman** [coturn/coturn#569]
I have a webrtc app, which works.

I want to understand what my stun/turn servers do.

My log config in turnserver.conf looks like this:

```
no-stdout-log
#log-file=/var/tmp/turn.log
#syslog
#simple-log
```

In the config it tells me that the default location for log files is:

`/var/log`

I checked this location and I have indeed some log files there, only for 3 days, which are super old. Maybe at the beginning I had a config, which worked. I dont know. I cant make it work right now.

[https://i.stack.imgur.com/yeOZQ.png](url)

One weird thing is, that all log files have the same content and size.

What am I doing wrong?

Maybe log files will only be created if errors occure? My app works without issues, maybe thats why I dont have log files?

> LOGS When the TURN Server starts, it makes efforts to create a log file turnserver_ < pid > .log in the following directories: /var/log /log/ /var/tmp /tmp

There are no logs in none of these folders. I tried now almost every single config combination and I also have restarted coturn everytime I changed the config.

**Comment by enoch85** [comment]
Same here, when I change the log file to `/var/log/whatever/turnserver.log` it doesn't write anything to it. I've tried setting `777` and `turnserver:turnserver` and `root:root` - nothing helps.

Guidance would be appriciated.

My config:
```
no-stdout-log
simple-log
log-file=/var/log/whatever/turnserver.log
```

**Comment by misi** [comment]
Check file in /var/log/turnserver/, /var/log, /var/tmp, /tmp 
https://github.com/coturn/coturn/blob/8c6b2f6135a51e7811191cd73b1bd2914f15aa6d/README.turnserver#L474

**Comment by enoch85** [comment]
@misi Sure, it's there, but shouldn't it be configurable to which ever path you want?

**Comment by jtague87** [comment]
I also seem to be having this problem. No matter what I do, I cannot get any connection logs to come through when testing from Trickle-ICE. Any insight @misi ?

**Comment by hbokh** [comment]
Used to work with Debian 9 Stretch `coturn 4.5.1.0-1~bpo9+1`.
Fails on Debian 10 Buster `coturn 4.5.1.1-1.1+deb10u1` with the same config.

**Comment by enoch85** [comment]
@hbokh Where was `syslog` commented? In `turnserver.conf?`

**Comment by hbokh** [comment]
> @hbokh Where was `syslog` commented? In `turnserver.conf?`

Yes, indeed.

**Comment by enoch85** [comment]
This is my config and I only use `simple-log`. Anything you propose I change?

```
# Generate $TURN_CONF
cat << TURN_CREATE > "$TURN_CONF"
listening-port=$TURN_PORT
fingerprint
use-auth-secret
static-auth-secret=$TURN_SECRET
realm=$TURN_DOMAIN
total-quota=100
bps-capacity=0
stale-nonce
no-multicast-peers
no-stdout-log
simple-log
log-file=$VMLOGS/turnserver.log
```

**Comment by misi** [comment]
@enoch85 I agree that it would be nice to make it configurable. I am not author of the original code just try to maintenance it, and unfortunately I have limited time for it. Please send a Pull Request and I am happy to accept it.
